scripts/update_schemas_version.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c
```

Comment:
This comment should be immediately before the `get_latest_release()` line.

Reply:
Moved it, thanks!